client-v2/src/components/inputs/richtext/setup.ts

```diff
@@ -14,18 +13,13 @@ import { history } from 'prosemirror-history';
 const createBasePlugins = (s: Schema) => {
   const plugins = [
     keymap(buildKeymap(s, {}, {})),
-    keymap(baseKeymap),
```

Comment:
Was `baseKeymap` just not necessary?

Reply:
Yeah, it was importing (and potentially overriding) bindings that we were declaring ourselves.